test/aria/widgets/container/dialog/modalMaskSize/DialogModalMaskSizeTestCase.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2017 Amadeus s.a.s.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+Aria.classDefinition({
+    $classpath : "test.aria.widgets.container.dialog.modalMaskSize.DialogModalMaskSizeTestCase",
+    $extends : "aria.jsunit.TemplateTestCase",
+    $dependencies: ["aria.utils.Json"],
+    $constructor : function () {
+        this.$TemplateTestCase.constructor.call(this);
+
+        this.data = {
+            boxWidth: 10,
+            boxHeight: 10
+        };
+        this.setTestEnv({
+            template : "test.aria.widgets.container.dialog.modalMaskSize.DialogModalMaskSize",
+            data : this.data,
+            iframe : true
+        });
+    },
+    $prototype : {
+        runTemplateTest : function () {
+            var self = this;
+            var testWindow = self.testWindow;
+            var document = testWindow.document;
+            var body = document.body;
+            var documentElement = document.documentElement;
+            var jsonUtils = testWindow.aria.utils.Json;
+            var domUtils = testWindow.aria.utils.Dom;
```

Comment:
Why not using the dependencies from the current context instead of using the ones from the test? (which are theoretically not ensured to be available, even if in this case we know they'll be)

I you keep your code like this, remove the `aria.utils.Json` dependency from the `$dependencies` field. If you change it instead, don't forget to add `aria.utils.Dom` to it.

Reply:
I have now added `aria.utils.Json` and `aria.utils.Dom` to the `$dependencies` array of the template to make sure that those dependencies are included in the test window, even though this is not really mandatory, because those classes are used in the template loading process (so we know that they will already be loaded).

Reply:
I have also removed `aria.utils.Json` from the `$dependencies` field of the test.